datascience/tables.py

```diff
@@ -266,12 +266,16 @@ def column(self, index_or_label):
         """
         return self._columns[self._as_label(index_or_label)]
 
-    # Deprecated
-    def values(self, label):
-        """Returns the values of a column as an array. [Deprecated]"""
-        warnings.warn("values is deprecated; use column", FutureWarning)
-        return self.column(label)
+    @property
+    def values(self):
+        dtypes = [col.dtype for col in self.columns]
+        if len(set(dtypes)) > 1:
+            dtype = object
+        else:
+            dtype = None
+        return np.array(self.rows, dtype=dtype)
 
+    # Deprecated
```

Comment:
Is this supposed to be deprecated, or did the comment just move from the old values function?